packages/pnpm/test/recursive/outdated.ts

```diff
@@ -50,6 +50,72 @@ test('pnpm recursive outdated', async (t: tape.Test) => {
 
     t.equal(result.status, 0)
 
+    t.equal(normalizeNewline(result.stdout.toString()), stripIndent`
+    ┌───────────────────┬─────────┬────────┬──────────────────────┐
+    │ Package           │ Current │ Latest │ Dependents           │
+    ├───────────────────┼─────────┼────────┼──────────────────────┤
+    │ is-negative       │ 1.0.0   │ 2.1.0  │ project-2            │
+    ├───────────────────┼─────────┼────────┼──────────────────────┤
+    │ is-negative (dev) │ 1.0.0   │ 2.1.0  │ project-3            │
+    ├───────────────────┼─────────┼────────┼──────────────────────┤
+    │ is-positive       │ 1.0.0   │ 3.1.0  │ project-1, project-3 │
+    └───────────────────┴─────────┴────────┴──────────────────────┘
+    ` + '\n')
+  }
+
+  {
+    const result = execPnpmSync('recursive', 'outdated', 'is-positive')
+
+    t.equal(result.status, 0)
+
+    t.equal(normalizeNewline(result.stdout.toString()), stripIndent`
+    ┌─────────────┬─────────┬────────┬──────────────────────┐
+    │ Package     │ Current │ Latest │ Dependents           │
+    ├─────────────┼─────────┼────────┼──────────────────────┤
+    │ is-positive │ 1.0.0   │ 3.1.0  │ project-1, project-3 │
+    └─────────────┴─────────┴────────┴──────────────────────┘
+    ` + '\n')
+  }
+})
+
+test('pnpm recursive outdated: show details', async (t: tape.Test) => {
```

Comment:
I would unite these to tests. You can run both `pnpm -r --long outdated` and `pnpm -r outdated` on the same project. This will reduce the time that is needed to bootstrap the fixture.